### ISSUE TITLE
The PassThru Object CodeCoverage property only has a line number when a command may span line

### ISSUE THREAD

**Issue by TravisEz13** [pester/Pester#727]
The `hitCommands` and `missedCommands properties` of the `CodeCoverage` Property returned on the object when `-PassThru` is specified online contains one line number when a command may span multiple lines.

This can produce results like this (color code, white = not code, red= missed, green = hit)
https://codecov.io/gh/PowerShell/PSDscResources/src/aa96caa838845ffad9658369d196a778c22b9931/DscResources/MSFT_UserResource/MSFT_UserResource.psm1#L1149

![image](https://cloud.githubusercontent.com/assets/10873629/23003361/325e4012-f3a5-11e6-92d1-e3fd02398607.png)

**Comment by nohwnd** [comment]
@Thilas after #1120 and #1920 was merged do you think this is still an issue? Could you test it maybe? :)  (The changes are part of master, but are not released yet.)

**Comment by nohwnd** [comment]
@Thilas Thanks. 
Marked this as help wanted. The goal here would be to test this to see if we can get all the lines to turn green somehow.

**Comment by Thilas** [comment]
@nohwnd, working on other PR, I figured out it was not that complex to do this so here is a new PR: #1190.

**Comment by Thilas** [comment]
@nohwnd, I guess this issue can be closed now.

**Comment by nohwnd** [comment]
Should be fixed by #1190
